user-guide/Advanced_Modules/SRM/Advanced_SRM_settings.md

```diff
@@ -2,24 +2,34 @@
 uid: Advanced_SRM_settings
 ---
 
-# Advanced SRM settings
+# Function resources
 
-From DataMiner 9.5.7 onwards, advanced SRM settings can be configured in the file *Config.xml*, in the folder *C:\\Skyline DataMiner\\ProtocolFunctionManager*. This file contains the following tags:
+From DataMiner 9.5.7 onwards, some advanced settings related to function DVEs linked with a function resource can be configured in the file *Config.xml*, in the folder *C:\\Skyline DataMiner\\ProtocolFunctionManager*.
 
-| Tag | Description |
-|--|--|
-| ActiveFunctionResourcesThreshold | Used to configure the number of virtual function resources that can be active at the same time before a disabling of virtual function resources is triggered. Set this to an integer representing the maximum number of active virtual function resources. By default: 100. |
-| FunctionHysteresis | Can be used to keep virtual function resources from being disabled if they will be used again soon. Set this to the number of seconds in the future that the system has to check whether a virtual function resource is in use, in order to determine whether it should be disabled. By default, up to DataMiner 9.5.12, this is set to *PT0S*, which means no hysteresis is applied. From DataMiner 9.5.13 onwards, the default value is 10 minutes. |
+This file contains the following tags:
 
-> [!NOTE]
-> From DataMiner 9.5.8 onwards, the hysteresis functionality is expanded to also include the preloading of virtual function resources. This means that if, for example, function hysteresis is set to 10 minutes, any disabled virtual function resources that are required for a booking instance will be enabled 10 minutes before the start of the booking instance.
+- **ActiveFunctionResourcesThreshold**: Used to configure the number of function DVEs that can be active at the same time. Their deactivation is scheduled after a booking has finished. They only get deactivated if the threshold is reached. The default threshold is 100.
+
+  The threshold will not act as a limit. The function DVEs assigned to ongoing bookings and bookings scheduled within the hysteresis time (see *FunctionHysteresis* below) will not get deactivated automatically.
+
+- **FunctionHysteresis**: This hysteresis is considered when the deactivation of function DVEs gets triggered (see *ActiveFunctionResourcesThreshold* above), to keep them from being disabled if they will be used again soon. This time span will be added to the start time of bookings and used to check whether a function resource is in use, in order to determine whether it should be disabled.
```

Comment:
Could we add more information about the format of this value (and that of the FunctionActivationTimeout setting)? Based on the example, this doesn't seem to be an obvious thing.

Reply:
Both are .NET timespans. In XML that translates to a [ISO8601 duration](https://en.wikipedia.org/wiki/ISO_8601#Durations). Not sure if we should go into detail. The example with code should be clear enough.